sopel/modules/url.py

```diff
@@ -231,6 +249,13 @@ def get_hostname(url):
     return hostname
 
 
+def get_tinyurl(url):
+    """ Returns a shortened tinyURL link of the URL. """
+    tinyurl = "http://tinyurl.com/api-create.php?url=%s" % url
```

Comment:
No particular reason; I've swapped it to HTTPS because it's a good idea. I've also updated the script to return HTTPS links since it turns out TinyURL's API returns HTTP links regardless if you call it using HTTP or HTTPS.

Reply:
That's very silly of TinyURL, but I appreciate that you took care of it!